src/index.njk

```diff
@@ -220,7 +230,6 @@ eleventyNavigation:
       <span>
         Hang out with us and others like you on
         <a href="https://discord.com/invite/8caJbAGNwY">Discord</a>,
-        <a href="https://web.libera.chat/?channel=#airyx">IRC</a>, or
```

Comment:
Take out the text "We’ve got a Discord, IRC, and Matrix. Chats are bridged. There's also a Telegram group (not currently bridged)." above. Move the note about Telegram not bridged to after the Telegram link.